hosts/Configuration/IdentityServerExtensions.cs

```diff
@@ -35,7 +35,6 @@ internal static WebApplicationBuilder ConfigureIdentityServer(this WebApplicatio
             .AddInMemoryIdentityResources(Resources.IdentityResources)
             .AddInMemoryApiScopes(Resources.ApiScopes)
             .AddInMemoryApiResources(Resources.ApiResources)
-            //.AddStaticSigningCredential()
```

Comment:
No need to fix this in both main and configuration hosts. The configuration host doesn't need this.